### ISSUE TITLE
Add language to app.js? 

### ISSUE THREAD

**Issue by web0174** [rothsandro/eleventy-notes#45]
Hello, can you add language to app.js?

**`base.layout.njk`**

```njk 
<html lang="{{ app.language }}" data-theme="system"> 
```

**`app.js`**

```js
module.exports = defineConfig({
  language: "it",
})
```

That's what I did, but I don't know if it's the best way, that's up to you. 
Now in the html I see my language.
_Sorry for my English, but it's not my language. Thank you! 😀_

```html

<html lang="it" data-theme="system" class="supports-js"><head>

```

**Comment by rothsandro** [comment]
Sure, I can add a config option for the language. Is all your content in Italian? Not just the markdown content but sections / groups in the sidebar (e.g. folders), tags and custom properties.

In addition, we'll need to add `lang="en"` on UI elements of Eleventy Notes (search, bookmarks, ...) as they are always in English.

**Comment by web0174** [comment]
> Tutti i tuoi contenuti sono in italiano? 
> Non solo il contenuto del markdown ma anche sezioni/gruppi nella barra laterale (ad esempio cartelle), tag e proprietà 
> personalizzate

Thanks for your answer 🙏. For now I haven't translated almost anything because I'm doing a bit of testing and comparing between 4 different ssg. Thanks!! 👋🏻

**Comment by rothsandro** [comment]
**Feature shipped** 🎉 Eleventy Notes v0.20.0 adds a new `lang` config option
